### PR TITLE
Enable plugin portal mirrors for integration tests by default

### PR DESCRIPTION
**Pull request by blindpirate**

### CODE REVIEW

**Comment by blindpirate** [comment]
@bot-gradle test this

**Comment by blindpirate** [comment]
@bot-gradle test this

**Comment by blindpirate** [comment]
@bot-gradle test rfn

File: subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/catalog/AbstractVersionCatalogIntegrationTest.groovy [hunk 1]
```diff
@@ -43,5 +43,6 @@ abstract class AbstractVersionCatalogIntegrationTest extends AbstractHttpDepende
         """
         resolve.expectDefaultConfiguration("runtime")
         resolve.prepare()
+        executer.withPluginRepositoryMirrorDisabled() // otherwise the plugin portal fixture doesn't work!
```
Comment (wolfs) [inline subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/catalog/AbstractVersionCatalogIntegrationTest.groovy:46]: Should we maybe add this method to the plugin portal fixture or have the plugin portal fixture disable the repository mirrors automatically?

**Review by wolfs — COMMENTED** [review]
LGTM! I left one comment.

File: subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/catalog/AbstractVersionCatalogIntegrationTest.groovy (hunk 1 above)
Comment (blindpirate) [reply subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/catalog/AbstractVersionCatalogIntegrationTest.groovy:46]: Which plugin portal fixture?

**Comment by blindpirate** [comment]
@bot-gradle test rfr

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Pull request merged**